build/Chart.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v2
+name: openfunction
+description: A Helm chart for OpenFunction on Kubernetes
+# Specify the Kubernetes version range that we support.
+# We allow pre-release versions for cloud-specific Kubernetes versions such as  v1.21.5-gke.1302 or v1.18.9-eks-d1db3c
+kubeVersion: ">=v1.20.0-0"
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "0.6.0"
+keywords:
+  - faas
+  - Serverless
+  - Kubernetes
+sources:
+  - https://github.com/OpenFunction/OpenFunction
+maintainers:
+  - email: wangyifei@kubesphere.io
+    name: wangyifei
+dependencies:
+  - name: dapr
+    version: 1.5.1
+    repository: "file://dapr"
+    condition: dapr.enabled
+  - name: knative-serving
+    version: 0.6.0
```

Comment:
knative's version is not 0.6.0

Reply:
This is what we decided last time we discussed it in discord

Reply:
![image](https://user-images.githubusercontent.com/3085804/172287903-35c9405d-f1cf-428b-98e5-b48e251b3f1a.png)